osp-workshop-2024.sh

```diff
@@ -66,7 +70,20 @@ EOF
     fi
 }
 
-while getopts "b:dKs:i:p:c:o:n" opt_key; do
+function configure_ssh_jump_host() {
+    grep -q $jump_host $SSH_CONFIG
```

Comment:
Would be better to pass a parameter to the function to make it more generic and less dependent on a global variable:
```
    local jump_host=$1
    ....

L16 in the other file: configure_ssh_jump_host $jump_host
```

Reply:
Done